crc/services/workflow_processor.py

```diff
@@ -428,3 +428,7 @@ def find_task_and_field_by_field_id(self, field_id):
                         return spiff_task, field
         raise ApiError("invalid_field",
                        "Unable to find a task in the workflow with a lookup field called: %s" % field_id)
+
+    @staticmethod
+    def get_script_engine():
```

Comment:
@danfunk this method is not used, did you intend to replace it within `evaluate_python_expression`?